pkg/skaffold/build/buildpacks/init_test.go

```diff
@@ -32,17 +32,27 @@ func TestValidate(t *testing.T) {
 	}{
 		{
 			description:   "NodeJS",
-			path:          "path/to/package.json",
+			path:          filepath.Join("path", "to", "package.json"),
```

Comment:
just for completeness add corner test cases for
`go.mod`, `package.json` -> i.e. no filepath.Base = empty string
and `somedir/vendor/go.mod` -> i.e vendor not first parent.